force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -162,7 +206,8 @@ public with sharing class VariablesDataTypesOperators {
     Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
```

Comment:
Try to make the MILLISECONDS_PER_MINUTE  static and final and assign it the value 6000 
Then you can remove the variable milliseconds